test/test_input_output_aliases.py

```diff
@@ -143,6 +144,28 @@ def try_grad_accum(model, device, train_x, train_label, accum_steps):
         alias_count == 1.0
     ), f"Expect 1 input-output alias pair for gradient accumulation, got {alias_count}"
 
+  def test_xm_save_no_aliasing(self):
```

Comment:
I think this test is added in https://github.com/pytorch/xla/pull/8467/files already

Reply:
rebased.